examples/examples.tf

```diff
@@ -0,0 +1,18 @@
+module "github-iam" {
```

Comment:
Related to the examples let's create also those files:
- `versions.tf`
- `outputs.tf`
-  `providers.tf`
- `main.tf `(renamed from examples.tf)

Inside of the `versions.tf` you can put something like this:
```
terraform {
  required_version = "<latest terraform version>"

  required_providers {
    github = {
      source = "integrations/github"
      version = "5.4.0"
    }
  }
}
```
Inside of the  `providers.tf` smth like this:
```
provider "github" {
  # Configuration options
}
```

The idea with the examples is that, if I want to use this module I can take the code from examples and run it.

Reply:
Done